db/migrate/20201012013526_add_author_to_pull_request.rb

```diff
@@ -0,0 +1,5 @@
+class AddAuthorToPullRequest < ActiveRecord::Migration[6.0]
+  def change
+    add_column :pull_requests, :author, :string
```

Comment:
maybe this can be called `author_name`? `author` alone can mean an object, an id, a name